saas/axops/src/ui/src/app/views/+user-management/user-profile/user-profile.component.ts

```diff
@@ -27,8 +27,7 @@ export class UserProfileComponent implements OnInit, OnDestroy, LayoutSettings,
         private usersService: UsersService,
         private authenticationService: AuthenticationService,
         private activatedRoute: ActivatedRoute,
-        private utils: UserUtils,
-        private sharedService: SharedService) {
+        private utils: UserUtils) {
     }
 
     public get layoutSettings(): LayoutSettings {
```

Comment:
Thank you for explanation @wokeGit ! Now I understand why SharedService was created. I've updated LayoutComponent so layout setting might be changed after navigation end event: https://github.com/argoproj/argo/pull/24/commits/f2167b62c1d80b8e839cb12605f8fb69afefa7fb  

Please take a look